adwords_client/adwords_api/common.py

```diff
@@ -184,4 +200,10 @@ def __init__(self, service):
         self.operations = []
 
     def add_operation(self, operation):
+        if self.operations:
+            if operation.get('xsi_type', None) != self.operations[-1].get('xsi_type', None):
```

Comment:
When you use `.get()` and the key does not exist, it returns `None` by default (https://docs.python.org/3/library/stdtypes.html#dict.get). This line could be re-written as 
```if operation.get('xsi_type') != self.operations[-1].get('xsi_type'):```